arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1936 @@
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/IotHubKeys/{keyName}/listkeys": {
+      "post": {
+        "tags": ["POST"],
+        "summary": "Get the Key given a specific KeyName for an IotHub",
+        "description": "Get the Key given a specific KeyName for an IotHub",
+        "operationId": "IotHubResource_GetKeysForKeyName",
+        "consumes": [],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "Name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "keyName",
+          "in": "path",
+          "description": "Name of the key.",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SharedAccessSignatureAuthorizationRule"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/exportDevices": {
+      "post": {
+        "tags": ["POST"],
+        "summary": "Export all the Devices in an IotHub",
+        "description": "Export all the Devices in an IotHub",
+        "operationId": "IotHubResource_ExportDevices",
+        "consumes": ["application/json", "text/json"],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "Name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "exportDevicesRequest",
+          "in": "body",
+          "description": "The export devices request.",
+          "required": true,
+          "schema": {
+            "$ref": "#/definitions/ExportDevicesRequest"
+          }
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
```

Comment:
Missing description. (Descriptions are important, they are used to generate the help documentation for the APIs)